docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,66 @@
+==============
+Atop connector
+==============
+
+Overview
+--------
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System & Process Monitor) Linux server performance analysis tool.
```

Comment:
link to  https://www.atoptool.nl/  .. provided thats the right website

Reply:
I'm not that sure whether this is the link to the company behind ``atop``

Reply:
That is the correct one. See the "External resources" in the right hand sidebar of https://packages.debian.org/sid/atop